docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -93,7 +93,7 @@ Even though Kubernetes provides a lot of functionality, there are always new sce
 
 Additionally, the [Kubernetes control plane](/docs/concepts/overview/components/) is built upon the same [APIs](/docs/reference/api-overview/) that are available to developers and users. Users can write their own controllers, such as [schedulers](https://github.com/kubernetes/community/blob/{{page.githubbranch}}/contributors/devel/scheduler.md), with [their own APIs](https://github.com/kubernetes/community/blob/{{page.githubbranch}}/contributors/design-proposals/extending-api.md) that can be targeted by a general-purpose [command-line tool](/docs/user-guide/kubectl-overview/).
 
-This [design](https://github.com/kubernetes/community/blob/{{page.githubbranch}}/contributors/design-proposals/principles.md) has enabled a number of other systems to build atop Kubernetes.
+This [design](https://git.k8s.io/community/contributors/design-proposals/architecture/principles.md) has enabled a number of other systems to build atop Kubernetes.
```

Comment:
I prefer `github.com/kubernetes` to `git.k8s.io` :)

Other similar places in the PR too.

Reply:
As per https://github.com/kubernetes/community/pull/1080#discussion_r139492007 , `git.k8s.io` would be ok.